R/initModel.R

```diff
@@ -60,13 +62,20 @@ initModel <- function(config = NULL,
     stop("sendToSlurm is TRUE, but SLURM is not available. Stopping.")
   }
 
-  # Check if an already existing path was given
-  if (!is.null(path) && file.exists(path)) {
-    message("Given path already exists. Restarting on this path.")
-    if (is.null(restart)) {
+  # Check if this is a restart run and determine the path to be restarted
+  if (isTRUE(restart) || is.character(restart)) {
```

Comment:
Am I right that we'd never expect a `config` given to a restart run?
```suggestion
  if (isTRUE(restart) || is.character(restart)) {
    if (!is.null(config)) {
      stop("A restart run cannot take a config.")
    }
```

Reply:
Actually, a similar check is found some lines below, but currently it only generates a warning. In my view this might make sense, because maybe a user included the config argument by accident, e.g. from an earlier call to initModel. But if you prefer to throw an error, we can also adapt this.

Reply:
You're right, I missed that. Then let's keep the warning for now.